apollo-spaceport/src/server.rs

```diff
@@ -298,6 +316,7 @@ impl ReportSpaceport {
         // that we are adding a lot of data for transfer. It is derived
         // empirically from load testing.
         let total = self.total.fetch_add(1, Ordering::SeqCst);
+        // TODO: check tick ?
```

Comment:
yes, we should check a timeout here, otherwise no data will be sent until the batch is full

Reply:
that's not a great experience when in development because there can be very few queries after each restart

Reply:
Looks like it's working, I removed the comment: [0def32e](https://github.com/apollographql/router/pull/898/commits/0def32e3b9c81b265027cb8fa5afb90e4cf0532b)